content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
```

Comment:
Worth highlighting it relies on the same Library than is used by the APM suite, thus not requiring teams to deploy and maintain yet another one